localization/languages/pt-BR.json

```diff
@@ -180,7 +180,7 @@
         "appMenuAlwaysOnTop": null, //missing translation
```

Comment:
"appMenu Always On Top", something with -> "fixed menu always on top?

Reply:
The original string is ["Always on Top"](https://github.com/minbrowser/min/blob/master/localization/languages/en-US.json#L180). Enabling this setting makes Min stay on top of all other open windows, even when it isn't the currently-focused window.